blueocean-dashboard/src/main/js/components/RunDetailsHeader.jsx

```diff
@@ -124,7 +124,13 @@ class RunDetailsHeader extends Component {
             <div className="u-label-value" title={commitLabel + ': ' + commitIdString}>
                 <label className={labelClassName}>{ commitLabel }:</label>
                 <span className="commit">
```

Comment:
@NicuPascu to test this correctly:

1. Split this code for displaying commit IDs out into something like `CommitIdCell.jsx`
```
export default class CommitIdCell extends Component {
  render() { 
    return (<span className="commit">...</span>);
  }
}
```
2. Write an [enzyme test](http://airbnb.io/enzyme/docs/api/)
import React from 'react';
import { expect } from 'chai';
import { render } from 'enzyme';
import CommitIdCell from '../../main/js/components/CommitIdCell.jsx"';

describe('CommitIdCell', () => {

    it('displays commit ID with link', () => {
        const run =  { ... mocked run object goes here ... };
        const cell = render(<CommitIdCell run={run} t={t} />);
        expect(cell.text()).to.equal(... something ...); // you will have to change this to test the rendered output
    });


There is a good example in `RunMessageCell.jsx` and `run-message-cell-spec.js` on how to write these tests